packages/css/src/Title/Title.stories.ts

```diff
@@ -7,16 +7,33 @@ const meta: Meta = {
 
 export default meta;
 
+const MODIFIERS = ["", "content", "bigTitle"];
+
 export const Default: StoryObj = {
   render: (args) => {
-    const title = document.createElement("h1");
+    const title = document.createElement(args.heading);
     title.innerHTML = args.text;
-    title.className = args.classModifier ? args.classModifier : "";
+
+    title.className += [
+      "af-title",
+      args.classModifier ? `af-title--${args.classModifier}` : "",
+    ].join(" ");
 
     return title;
   },
   args: {
     text: "Sample Title",
-    classModifier: "af-title",
+    heading: "h2",
+    classModifier: "",
+  },
+  argTypes: {
+    classModifier: {
+      options: MODIFIERS,
+      control: { type: "select" },
```

Comment:
Il faut utiliser le "multi-select" plutôt que le select car on peut avoir plusieurs éléments dans le classModifier.
cf https://github.com/AxaFrance/design-system/blob/main/packages/css/src/Form/Text/Text.stories.ts#L52

Reply:
@GuillaumeKESTEMAN  in your case , vous pouvez mettre disabled & required ensemble, dans le cas du Title je ne peux pas mettre par exemple (content & bigTitle) ensemble

Reply:
Ha d'accord, autant pour moi 👍